src/scripts/main.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+const notification = document.createElement('div');
+
+document.body.appendChild(notification);
+
+let isResolved = false;
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    if (!isResolved) {
+      isResolved = true;
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    if (!isResolved) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('mousedown', (e) => {
```

Comment:
Don't need to use mousedown event, need to use only click and contextmenu events